CONTRIBUTING.md

```diff
@@ -9,24 +9,47 @@ Follow these instructions for contributing new recipes. The Goal headers indicat
 #### Goal 1: Claim and Start Working on a Recipe Request
 
 1. Search currently [open "Recipe Request" issues](https://github.com/JordanMartinez/purescript-cookbook/issues?q=is%3Aissue+is%3Aopen+label%3Arecipe-request).
-1. If your recipe idea is unique, open a new ["Recipe Request" issue](https://github.com/JordanMartinez/purescript-cookbook/issues/new?assignees=&labels=recipe-request&template=recipe-request.md&title=)
+1. (optional) If your recipe idea is unique, open a new ["Recipe Request" issue](https://github.com/JordanMartinez/purescript-cookbook/issues/new?assignees=&labels=recipe-request&template=recipe-request.md&title=). This step aims to prevents two people from working on the same recipe.
```

Comment:
We talked about this before, and I can split this out to another PR for further discussion.
I think making this mandatory adds unnecessary overhead. Proposing we let this be optional until recipe collisions become a problem.

Reply:
I'm good with this change. Name collisions haven't been an issue yet.